### DISCUSSION TITLE
Questions regarding custom config and styling

### DISCUSSION THREAD

**Discussion by pietz** [picocss/pico#473] [Help]
I love the concept of PicoCSS but I'm not a big fan of its default styling. Especially on desktop the components are so incredible large that it looks almost toy-ish in my eyes. I know, Pico supports a lot of customizing options but it appears a bit overwhelming for a pico newbie like me.

1) Is there a central setting that help me adjust components becoming very big on desktop views?

2) Has anyone created a config that makes PicoCSS appear more like Tailwind UI or Shadcn UI?

Thanks

**Comment by candidia** [comment]
I'm replacing the pico tokens with tailwind ones manually in my project
